packages/x-data-grid-premium/src/components/GridAggregationHeader.tsx

```diff
@@ -40,11 +39,15 @@ const GridAggregationFunctionLabel = styled('div', {
   overridesResolver: (_, styles) => styles.aggregationColumnHeaderLabel,
 })<{ ownerState: OwnerState }>(({ theme }) => {
   return {
+    // @ts-ignore `@mui/material` theme.typography does not exist
```

Comment:
All the `@ts-ignore` will be removed by my suggestion https://github.com/mui/mui-x/pull/10498/files#r1778208301